cmd/chisel/cmd_help.go

```diff
@@ -159,7 +159,7 @@ var helpCategories = []helpCategory{{
 }, {
 	Label:       "Action",
 	Description: "make things happen",
-	Commands:    []string{"cut"},
+	Commands:    []string{"cut", "find"},
```

Comment:
Btw, I have the same comment here as in #101 -> this should probably be in the "Basic" label

Reply:
Moved in https://github.com/canonical/chisel/pull/99/commits/03f077a80a80634fa510b3520f50d95d749c86f0. Thanks.